packages/runtime/runtime-definitions/src/test/types/validateRuntimeDefinitionsPrevious.generated.ts

```diff
@@ -479,6 +479,7 @@ declare function get_old_InterfaceDeclaration_IFluidDataStoreContext():
 declare function use_current_InterfaceDeclaration_IFluidDataStoreContext(
     use: TypeOnly<current.IFluidDataStoreContext>);
 use_current_InterfaceDeclaration_IFluidDataStoreContext(
+    // @ts-expect-error compatibility expected to be broken
     get_old_InterfaceDeclaration_IFluidDataStoreContext());
 
```

Comment:
i don't think this is a valid change to make in main without my recommendations, as it breaks a prod interface.